src/js/components/TrackingProgress.jsx

```diff
@@ -0,0 +1,51 @@
+import useAirtableCMS from "../hooks/useAirtableCMS";
+import Markdown from "react-markdown";
+import remarkGfm from "remark-gfm";
+
+const TrackingProgress = () => {
+  const trackingProgress = useAirtableCMS({
+    baseID: "app1YqNgXXkVH04nO",
+    tableName: "Tracking Progress",
+    keyField: "section",
+    fieldMapping: {
+      section: "Section",
+      content: "Content",
+    },
```

Comment:
No changes required for this PR, but I just realized we'll want to update the CMS to include an "Order" column which we can use to add a `sortBy` function here, to make sure the sections show up in the order we want. I'll add a ticket for this in the backlog.